tests/e2e/utils/helpers.ts

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios'
+import config from '../config'
+import _path from 'path'
+import * as fs from 'fs'
+
+const uploadedFiles: string[] = []
+
+const getWebDavFilePath = (user, filename) => {
+  return _path.join('remote.php/dav/files', user, filename)
+}
+
+const getWebDavTrashbinPath = (user) => {
+  return _path.join('remote.php/dav/trash-bin', user)
+}
+
+const makeApiRequest = ({ method, path, data = null }): Promise<any> => {
+  try {
+    const headers = {
+      Authorization: `Basic ${Buffer.from(`${config.adminUser}:${config.adminPassword}`).toString('base64')}`
+    }
+    return axios({
+      method,
+      url: _path.join(config.baseUrlOcis, path),
+      headers,
+      data
+    })
+  } catch (error) {
+    throw new Error(error.message)
+  }
+}
+
+export const uploadFile = async (filename: string): Promise<void> => {
+  const fileUploadUrl = getWebDavFilePath(config.adminUser, filename)
+  const filePath = _path.join(config.assets, filename)
+  const fileContent = fs.readFileSync(filePath)
+  const response = await makeApiRequest({
+    method: 'PUT',
+    path: fileUploadUrl,
+    data: fileContent
+  })
+  if (response.status !== 201) {
+    throw new Error(`Failed to upload file ${filename}`)
+  }
+  uploadedFiles.push(filename)
+}
+
+const deleteFile = async (filename): Promise<void> => {
+  return await makeApiRequest({
+    method: 'DELETE',
+    path: getWebDavFilePath(config.adminUser, filename)
+  })
+}
+
+export const deleteAllFiles = async (): Promise<void> => {
+  while (uploadedFiles.length > 0) {
+    await deleteFile(uploadedFiles.pop())
+  }
+}
+
+export const emptyTrashbin = async (): Promise<void> => {
+  return await makeApiRequest({
+    method: 'DELETE',
+    path: getWebDavTrashbinPath(config.adminUser)
+  })
```

Comment:
status code not checked ?

Reply:
Similar here, the code should check that the HTTP status is OK, and if not, then fail the test. That helps someone running test scenarios to have a clue about what went wrong between scenarios, that might effect the next scenario.

Reply:
status code check has been added